.github/workflows/fetch_translations.yml

```diff
@@ -0,0 +1,34 @@
+name: Fetch Lokalize translations
+on:
+  workflow_dispatch:
+  schedule:
+    # Every Monday at 10 AM East Coast time / 7 AM West Coast time / 2 PM UTC
+    - cron: "0 14 * * 1"
+jobs:
+  fetch-lokalize-translations:
+    runs-on: macos-latest
+    steps:
+      # Check out the current state
+      - uses: actions/checkout@v3
+
+      # Fetch translations
+      - name: Run build script
+        run: cd scripts && ./localize.sh
+        shell: bash
+        env:
+          LOKALISE_API_TOKEN: ${{ secrets.LOKALISE_API_TOKEN }}
+
+      # Create timestamp
+      - name: Get timestamp
+        id: timestamp
+        run: echo "TIMESTAMP=$(date +'%Y%m%d-%H%M%S')" >> $GITHUB_OUTPUT
+
+      # Create the pull request
+      - name: Create Pull Request
```

Comment:
What happens if there's nothing to update?

Reply:
Then it won’t create a pull request.